src/OAuthClient/BibClient.php

```diff
@@ -26,17 +40,20 @@ public static function getBibByIdAndSource($bibId = '', $nyplSource)
         $response = json_decode((string)$response->getBody(), true);
 
         APILogger::addDebug(
-            'Retrieved bib by id and source',
+            'Retrieved bibs by ids',
             $response['data']
         );
 
         // Check statusCode range
         if ($statusCode === 200) {
-            return new Bib($response['data']);
+            foreach ($response['data'] as $bib) {
```

Comment:
Hmm... isn't `$response['data']` the bibs that you need? I'm not sure why you need this logic. Maybe I'm missing something though!